release-notes/docs/3.7.0.md

```diff
@@ -0,0 +1,203 @@
+# 3.7.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Configurable hierarchies](#configurable-hierarchies)
+- [Includes a new default configuration](#includes-a-new-default-configuration)
+- [Optimised form HTML generation](#optimised-form-html-generation)
+- [Improved purging now done server-side](#improved-purging-now-done-server-side)
+- [Warn when replicating too many docs](#warn-when-replicating-too-many-docs)
+- [Performance](#performance)
+- [And more...](#and-more)
+
+## Known issues
```

Comment:
should this be breaking change instead of known issue? since it will not be fixed